src/coreclr/pal/src/misc/jitsupport.cpp

```diff
@@ -155,7 +155,7 @@ PAL_GetJitCpuCapabilityFlags(CORJIT_FLAGS *flags)
 {
     _ASSERTE(flags);
 
-#if defined(HOST_ARM64)
+#if defined(TARGET_ARM64)
```

Comment:
This is testing the host Arm64 HW capabilities. What was wrong with `HOST...`? `TARGET...` does not sound right.

Reply:
This is to compile JIT, so isn't JIT a target in this case?

Reply:
This should not be compiled into any cross-targeting JIT. It would not return meaningful results for the target.

Reply:
Yes, the whole method (not just the body) should be under `#if defined(HOST_ARM64) && defined(TARGET_ARM64)`. The method is arm64-specific and nobody should be calling it in cross-targeting cases.

Reply:
`#elif defined(TARGET_ARM64)` block towards the end should be deleted if you accept this feedback.

Reply:
It is actually called on x64 too when target is arm64. It then intentionally executes this:
```
    // Enable ARM64 based flags by default so we always crossgen
    // ARM64 intrinsics for Linux
    flags->Set(InstructionSet_ArmBase);
    flags->Set(InstructionSet_AdvSimd);
```

Reply:
TARGET refers to the architecture you expect to be generating code for or debugging. If this is meant to be running in ARM64 (regardless of the machine that compiles it), then HOST is the correct variant.

Reply:
I see this only called from https://github.com/dotnet/runtime/blob/4cd3e780c469e2416b6ebaba903dbea7c411a5b8/src/coreclr/vm/codeman.cpp#L1529. What is the configuration where this place is compiled with cross-targeting? There should not be any situations like that.

Reply:
I guess you are right, the comment is wrong then.

Reply:
I've updated the PR based on the feedback